doc/release-notes-1.12.rst

```diff
@@ -286,3 +286,520 @@ release:
 
 .. comment  List derived from GitHub Issue query: ...
    * :github:`issuenumber` - issue title
+
+* :github:`1420` - LXR for Zephyr
+* :github:`1582` - USB: Add support for MS OS Descriptors
+* :github:`1620` - BT 4.2 Controller-based link-layer privacy
+* :github:`1651` - ARC: remove tickless idle dependency on SW ISR table
+* :github:`1655` - clean up how internal APIs are used
+* :github:`1799` - Provide an interface for cpu/soc id and version
+* :github:`1882` - SMP - Multi-core
+* :github:`1896` - Thread Protocol
+* :github:`2001` - Add support in I2C shim driver for DMA transfer mode
+* :github:`2002` - Add support in SPI shim driver for DMA transfer mode
+* :github:`2341` - Thread requirements in RFC1122 
+* :github:`2342` - Thread requirements in RFC2460
+* :github:`2343` - Thread requirements in RFC4291
+* :github:`2344` - Thread requirements in RFC4443
+* :github:`2345` - Thread requirements in RFC4944
+* :github:`2346` - Thread Requirements on RFC6282
```

Comment:
I would change the message at the beginning of this section to:

"These GitHub issues were closed since the previous 1.11.0 tagged
 release:	 release:"

addressed might give the impression all issues/enhancements have been fixed or implemented, as we can see in those items related to Thread, they were closed as not relevant.

Reply:
Done